### DISCUSSION TITLE
[Question]: does Panda only allow color themes/modes on the `html` tag?

### DISCUSSION THREAD

**Discussion by caseybaggz** [chakra-ui/panda#2285] [Q&A]
Our preset uses the Multi-theme/mode example on the docs where we have the [following conditions](https://github.com/pluralsight/pando/blob/main/packages/panda-preset/src/conditions.ts#L6) which I assumed would allow changing themes within nested children via:

```jsx
<html data-theme="inkyBlue" data-color-mode="dark">
...
<SomeChild className="light">...</SomeChild>
```

However, this doesn't seem to be working, nor using `cx('light', css(...))`? Is there an improvement we could make in our conditions to achieve this result in Panda?

Thanks!

**Comment by caseybaggz** [comment]
Doing more testing found out this combination works, but would love if the above worked to keep the code cleaner:

This does work:
```jsx
<html data-theme="inkyBlue" data-color-mode="dark">
...
<SomeChild data-theme="inkyBlue" data-color-mode="light">...</SomeChild>
```

**Reply by caseybaggz** [reply]
I think I understand the logic stemming from the modes being Objects within the Theme property but definitely feel as though the first example should be possible?

**Reply by astahmer** [reply]
using `:not()` to prevent CSS order issues will solve it
```ts
lightMode:
        "[data-color-mode=light] &:not(:is([data-color-mode=dark], .dark)), &.light, .light &:not(:is([data-color-mode=dark], .dark))",
      darkMode:
        "[data-color-mode=dark] &:not([data-color-mode=light], .light), &.dark, .dark &:not(:is([data-color-mode=light], .light))",
```
https://play.panda-css.com/LDCX0ZHezC

this also uses `:is()` to specify multiple conditions but if possible I think it would be best to only rely on either data-attribute or className rather than mixing both
